server/routes/recipes.js

```diff
@@ -0,0 +1,87 @@
+const router = require('express').Router();
+const mongoose = require('mongoose');
+
+let Recipe = require('../models/recipe.js')
+
+router.route('/').get((req, res) => {
+  return res.send('test');
+});
+
+//Create Recipe
+router.route('/new_recipe').post((req, res) => {
+
+  const _id = new mongoose.Types.ObjectId;
+  const name = req.body.name;
+  const ingredients = req.body.ingredients;
+  const description = req.body.description;
+  const price = req.body.price;
+
+
+  //Checks for
+  if(name.length == 0){
+  	res.send("Recipe must have a name");
+  }
+  if(ingredients.length == 0){
+  	res.send("Must list ingredients")
+  }
+  if(description.length == 0){
+    res.send("Recipe must have a description");
+  }
```

Comment:
Small thing, but I think we can just do these if statements like `if(!name)` instead of `name.length === 0`. Also remember to use `===` instead of `==` if you're doing things like comparing the length to 0.